docs/database.rst

```diff
@@ -0,0 +1,40 @@
+rotki Database
+##############
+
+rotki uses two different sqlite databases, one with information about assets (global.db) and one with user information (rotkehlchen.db).
+The latter is encrypted using an extension called `SQLCipher <https://github.com/sqlcipher/sqlcipher>`__
+that provides transparent 256-bit AES full database encryption.
+
+Database Location
+=================
+
+The default database locations are:
+
+- **Linux**: ``~/.share/local/rotki/data``
+- **OSX**: ``??``
+- **Windows**: ``??``
```

Comment:
This is stated here https://rotki.readthedocs.io/en/latest/usage_guide.html#rotki-data-directory maybe you can just link it